tests/unit/test_charm.py

```diff
@@ -32,16 +32,6 @@ def setUp(self):
         self.addCleanup(self.harness.cleanup)
         self.harness.begin()
 
-    @patch("slurm_ops_manager.SlurmManager.hostname", return_val="localhost")
-    def test_hostname(self, hostname) -> None:
-        """Test that the hostname property works."""
-        self.assertEqual(self.harness.charm.hostname, hostname)
-
-    @patch("slurm_ops_manager.SlurmManager.port", return_val=12345)
-    def test_port(self, port) -> None:
-        """Test that the port property works."""
-        self.assertEqual(self.harness.charm.port, port)
-
```

Comment:
Why did you elect to remove these unit tests? This will decrease our total test coverage over the code base. I understand dropping out the test for the `port` property since it's now idempotent (will always return 6817), but we should at least test the `hostname` property since it relies on the socket module, string formatting, and an index look up.

We could likely just do a MagicMock for the result of `socket.gethostname()` to at least ensure that the string formatting is producing the results we expect. Let me know what your thoughts on this are.